price-server/src/app/error.rs

```diff
@@ -12,4 +12,7 @@ pub enum PriceAppError {
     SubscriberError(#[from] SubscriberError),
     #[error("PriceAppError - ExchangePriceCacheError: {0}")]
     ExchangePriceCacheError(#[from] ExchangePriceCacheError),
+
+    #[error("PriceAppError - DecimalError: {0}")]
+    DecimalError(#[from] rust_decimal::Error),
```

Comment:
I'm a bit surprised to see this error included. Where can it originate from? Can't we rule it out?

Reply:
It originated from get_cents_per_sat_exchange_mid_rate()
https://github.com/GaloyMoney/stablesats-rs/blob/d989a8715b60c12e241ff937057aca4eeb682855/price-server/src/app/mod.rs#L266

I fixed it by converting the decimal to sats before calling try_from.